data-prepper-plugins/otlp-source/README.md

```diff
@@ -0,0 +1,96 @@
+# OTLP Source
+
+This is a source which follows the [OTLP Protocol](https://opentelemetry.io/docs/specs/otlp/) and supports three endpoints for logs, metrics, and traces. It supports both `OTLP/grpc` and `OTLP/HTTP`.
+
+## Usages
+
+### Routing telemetry signal based on event type
+
+Each of the telemetry signals may be sent to different processors or sinks based on specific needs. The routing to downstream pipelines is based on meta-data routing.
+
+```yaml
+otel-telemetry-pipeline:
+  source:
+    otlp:
+      ssl: false
+  route:
+    - logs: 'getMetadata("eventType") == "LOG"'
+    - traces: 'getMetadata("eventType") == "TRACE"'
```

Comment:
Shouldn't the event type be `SPAN`?

Reply:
Ideally yes, but this is our current implementation: https://github.com/opensearch-project/data-prepper/blob/b42225043c774e5b6e33988db2cf55d79192a0b2/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/JacksonSpan.java#L75

Reply:
I see.